### PR TITLE
Add dynamic container and skeleton loader

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue [hunk 1]
```diff
@@ -19,7 +19,10 @@
               class="white">
               <v-flex d-flex xs12>
                 <v-card flat>
-                  <v-card-text class="subtitle-2 grey--text pa-2">{{ this.$t('exoplatform.wallet.title.walletBalanceTitle') }}</v-card-text>
+                  <v-card-text
+                    class="subtitle-2 grey--text pa-2">
```
Comment (boubaker) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue]: please avoid using Vuetify colors helpers and use eXo colors helpers to make it more `brandable`. Thanks.

Comment (SaraBoutej) [reply wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue]: Done!

**Review by boubaker — DISMISSED** [review]

File: wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue [hunk 2]
```diff
@@ -28,7 +31,10 @@
                 justify-center>
                 <v-card flat>
                   <v-card-text class="pa-2">
-                    <a :href="walletUrl" class="display-1 font-weight-bold big-number">{{ walletBalance }} Ȼ</a>
+                    <a
+                      :href="walletUrl"
+                      class="display-1 font-weight-bold big-number"
+                      :class="firstLoadingWalletBalance && 'skeleton-text skeleton-background skeleton-border-radius'">{{ walletBalance }} Ȼ</a>
```
Comment (boubaker) [inline wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue]: `:class` should go before `class` ;-)

Comment (SaraBoutej) [reply wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue]: Done!

**Review by boubaker — APPROVED** [review]

**Pull request merged**